reana_workflow_controller/rest/utils.py

```diff
@@ -398,6 +400,7 @@ def list_directory_files(directory, search=None):
                         ResourceUnit.bytes_, file_details.size
                     ),
                 ),
+                "path": directory,
```

Comment:
Maybe I miss something, why this line is needed here?

Reply:
It isn't, I added it to print the path of the files to check they where stored in the correct place. But is a better idea to check it making the query to the database. I have already deleted it